modules/templates/util_misc.go

```diff
@@ -74,27 +74,31 @@ func ActionIcon(opType activities_model.ActionType) string {
 	switch opType {
 	case activities_model.ActionCreateRepo, activities_model.ActionTransferRepo, activities_model.ActionRenameRepo:
 		return "repo"
-	case activities_model.ActionCommitRepo, activities_model.ActionPushTag, activities_model.ActionDeleteTag, activities_model.ActionDeleteBranch:
+	case activities_model.ActionCommitRepo:
 		return "git-commit"
-	case activities_model.ActionCreateIssue:
-		return "issue-opened"
-	case activities_model.ActionCreatePullRequest:
-		return "git-pull-request"
-	case activities_model.ActionCommentIssue, activities_model.ActionCommentPull:
-		return "comment-discussion"
+	case activities_model.ActionDeleteBranch:
+		return "git-branch"
 	case activities_model.ActionMergePullRequest, activities_model.ActionAutoMergePullRequest:
 		return "git-merge"
-	case activities_model.ActionCloseIssue, activities_model.ActionClosePullRequest:
+	case activities_model.ActionCreatePullRequest:
+		return "git-pull-request"
+	case activities_model.ActionClosePullRequest:
+		return "git-pull-request-closed"
+	case activities_model.ActionCreateIssue:
+		return "issue-opened"
+	case activities_model.ActionCloseIssue:
 		return "issue-closed"
 	case activities_model.ActionReopenIssue, activities_model.ActionReopenPullRequest:
 		return "issue-reopened"
+	case activities_model.ActionCommentIssue, activities_model.ActionCommentPull:
+		return "comment-discussion"
 	case activities_model.ActionMirrorSyncPush, activities_model.ActionMirrorSyncCreate, activities_model.ActionMirrorSyncDelete:
 		return "mirror"
 	case activities_model.ActionApprovePullRequest:
 		return "check"
 	case activities_model.ActionRejectPullRequest:
-		return "diff"
-	case activities_model.ActionPublishRelease:
+		return "file-diff"
+	case activities_model.ActionPublishRelease, activities_model.ActionPushTag, activities_model.ActionDeleteTag:
```

Comment:
Optimally, there would be a `tag deleted` icon, but I think if anything, we have to suggest that upstream.

Reply:
Same could be said about branch deletion. Not sure if it would be accepted upstream.

Reply:
I mean… I don't know why it shouldn't.
The only reasoning I can see is that no one has the time to implement it, but apart from that there's nothing speaking against it as it **is** needed.

Reply:
Oh.
![grafik](https://github.com/go-gitea/gitea/assets/51889757/6f498bf8-21d4-4beb-a002-c1d5a0b31946)

Reply:
They will accept a resonably articulated feedback type issue I assume.

Reply:
Just to add to this, they do... Once gave feedback on the site and they implemented it eventually... Tho, it can take a long time to be implemented (And I recall using bug report, but I assume the feedback one may work as well)